src/v/utils/log_hist.h

```diff
@@ -199,6 +199,8 @@ class log_hist {
      */
     seastar::metrics::histogram internal_histogram_logform() const;
 
+    seastar::metrics::histogram read_dist_histogram_logform() const;
```

Comment:
`log_hist` is designed to be an approximation of a histogram. Take the type;
```
using log_hist_internal = log_hist<std::chrono::microseconds, 26, 8ul>;
```
there will be 26 buckets each of which is an unsigned integer. Each bucket will end up as another metric series. So 26 in this case or 18 in the public histogram case.  for `log_hist_internal` the buckets are laid out like; 
```
[0, 8us), [8us, 16us), [16us, 32us), [32us, 64us)....[~1min, +inf)
```
when a duration is record the bucket whose range includes the duration is found. Then the int for that bucket is incremented.

From this we can see that the two existing resuables are great for recording latency since there is a lot of granularity around the 0 to 1s range. However, for this metric it seems we care a bit more about broader ranges like minutes or hours. So the existing resuables won't work as all those values will be recorded to the last bucket and the resulting histogram won't be very useful.

So to that ends I think what @WillemKauf has done here is fine. It gives up the granularity we have in the latency histograms around 0-1s in exchange for having more buckets over a longer range of time.

Reply:
We probably want to make the comments on the resuable types a bit more instructive. I.e, this is the histogram you want to use if you only care about accurately representing sub-second durations, minute/hour durations, day durations... etc.

Reply:
TDLR; the resuable types were only designed for sub-second latency durations. It makes sense to me to add more that more accurately represent other ranges of times folks may be interested in.